### ISSUE TITLE
Remove trailing semicolons in notebook cells

### ISSUE THREAD

**Issue by CarstZ** [dnanhkhoa/nb_black#11]
It seems that black by default deletes trailing semicolons in notebook cells. As these are commonly used to suppress the output of the cells, this can be a bit annoying.
Is there any possibility to tell nb_black / black to change this behaviour?

I know that I can just write None in the last line of each python cell but this makes the code kind of ugly.

Thanks in advance and kind regards,
Carsten

**Comment by graingert** [comment]
so black_nb could strip all trailing whitespace and trailing `;` before passing the code to black

then add 1 trailing `;` if it removed any `;`

**Comment by MarcoGorelli** [comment]
@czwilling in case it's useful to you, [nbqa](https://github.com/nbQA-dev/nbQA) preserves trailing semicolons when running `black` on notebooks

**Comment by fingoldo** [comment]
Indeed, I hate when the extension removes the semicolons that I put intentionally to hide garbage output from matplotlib charts. Any solution, please?

**Comment by MarcoGorelli** [comment]
I don't think this repo is maintained anymore, I'd suggest having a look at https://github.com/n8henrie/jupyter-black
